### PR TITLE
GG-19572: Add cluster id prototype.

### PR DESCRIPTION
**Pull request by smoldachev**

### CODE REVIEW

File: modules/web-console/web-console-server/src/test/java/org/apache/ignite/console/web/controller/ClusterInfoControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package org.apache.ignite.console.web.controller;
```
Comment (akuznetsov-gridgain) [inline modules/web-console/web-console-server/src/test/java/org/apache/ignite/console/web/controller/ClusterInfoControllerTest.java]: Missing license.

File: modules/web-console/web-console-server/pom.xml [hunk 2]
```diff
@@ -162,6 +162,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-test</artifactId>
```
Comment (akuznetsov-gridgain) [inline modules/web-console/web-console-server/pom.xml]: Why we need this dependency?

Comment (smoldachev) [reply modules/web-console/web-console-server/pom.xml]: For this amazing annotation @WithMockUser :)

File: modules/web-console/web-console-server/src/test/java/org/apache/ignite/console/web/controller/ClusterInfoControllerTest.java (hunk 1 above)
Comment (smoldachev) [reply modules/web-console/web-console-server/src/test/java/org/apache/ignite/console/web/controller/ClusterInfoControllerTest.java]: Fix

**Pull request merged**